pkg/apis/externaldns/types.go

```diff
@@ -289,7 +289,7 @@ var defaultConfig = &Config{
 	OVHApiRateLimit:             20,
 	PDNSServer:                  "http://localhost:8081",
 	PDNSAPIKey:                  "",
-	PDNSTLSEnabled:              false,
+	PDNSTLSDisabled:             false,
```

Comment:
```suggestion
	PDNSSkipTLSVerify:             false,
```

Reply:
Changed there and also in other places by [d0fabd0](https://github.com/kubernetes-sigs/external-dns/pull/3839/commits/d0fabd0695549a56c0a1a2f492cec288a749943a).